site/en/docs/extensions/mv3/faq/index.md

```diff
@@ -40,66 +40,40 @@ the browser.
-linux,stable,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-mac,dev,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-mac,beta,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-mac,stable,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-win,canary,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-win,dev,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-win,beta,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-win,stable,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-cros,dev,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-cros,beta,#.#.###.#,#.#.###.#,mm/dd/yy,mm/dd/yy,#####,#####,#####
-```
-
-Each line represents information about a different platform and channel combination. The listed
-platforms are `cf` (Google Chrome Frame), `linux`, `mac`, `win`, and `cros` (Google ChromeOS). The
-listed channels are `canary`, `dev`, `beta`, and `stable`. The two four-part numbers after the
-channel represent the current and previous versions of Chrome deployed to that platform-channel
-combination. The rest of the information is metadata about when the releases were first pushed, as
-well as revision numbers associated with each build.
+visit [chrome releases][34], which lists the current Chrome releases for each channel and platform.
 
 ## Capabilities {: #capabilities2 }
 
-### Can extensions make cross-domain Ajax requests? {: #faq-dev-02 }
+### Can extensions make cross-origin network requests? {: #faq-dev-02 }
 
-Yes. Extensions can make cross-domain requests. See [this page][35] for more information.
+Yes. Extensions can make cross-origin network requests. See [this page][35] for more information.
 
 ### Can extensions use 3rd party web services? {: #faq-dev-03 }
 
-Yes. Extensions are capable of making cross-domain Ajax requests, so they can call remote APIs
+Yes. Extensions are capable of making cross-origin network requests, so they can call remote APIs
 directly. APIs that provide data in JSON format are particularly easy to use.
 
 ### Can extensions encode/decode JSON data? {: #faq-dev-07 }
 
-Yes, modern JavaScript engines like Chrome's V8 have built-in support for
+Yes. Chrome's JavaScript engine, called "V8", has built-in support for
 [JSON.stringify][stringify] and [JSON.parse][parse] so you can use these
 functions in your extensions without including JSON libraries in
 your code.
 
 ### Can extensions store data locally? {: #faq-dev-08 }
 
-Yes, extensions can use [localStorage][37] to store string data permanently. Using Chrome's built-in
-JSON functions, you can store complex data structures in localStorage. For extensions that need to
-execute SQL queries on their stored data, Chrome implements [client side SQL databases][38], which
-may be used as well.
+Yes. Extensions provides a [storage API][63] designed specifically for extenions. You can also use
+the web platform's storage APIs: the [Web Storage API][37], and [IndexedDB][38]. If you use an
```

Comment:
This may confuse developers in thinking they can use Window.localStorage in a service worker.